classExample.rb

```diff
@@ -0,0 +1,34 @@
+class Rectangle
+   # constructor method
+   def initialize(w,h)
+      @width = w
+      @height = h
+   end
+
+   # accessor methods
+   def printWidth
```

Comment:
check naming conventions for method in ruby.
`print_width`

Reply:
Correct at all places. Naming conventions

Reply:
Variables are named as per naming conventions.